packages/react/src/themes/teams/components/Label/labelStyles.ts

```diff
@@ -4,17 +4,20 @@ import { pxToRem } from '../../../../lib'
 import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
 import { LabelProps } from '../../../../components/Label/Label'
 import { LabelVariables } from './labelVariables'
+import { getColorSchemeKey } from '../../colors'
 
 const labelStyles: ComponentSlotStylesInput<LabelProps, LabelVariables> = {
-  root: ({ props: p, variables: v, colors }): ICSSInJSStyle => {
+  root: ({ props: p, variables: v }): ICSSInJSStyle => {
+    const colors = v.colorScheme[getColorSchemeKey(p.color)]
```

Comment:
```suggestion
    const colors = getColorScheme(v.colorScheme, p.color)
```

Just suggestion :) Why we can grab it directly?

Reply:
I've introduced this change, looks much cleaner!